src/System.Devices.Gpio/Planning.md

```diff
@@ -0,0 +1,403 @@
+# Goals
```

Comment:
Why is this file here and not in https://github.com/dotnet/corefxlab/tree/master/docs/specs